src/components/FullWidthPromo/_full-width-promo.scss

```diff
@@ -0,0 +1,165 @@
+// ----------------------------------
+// UI Components
+// Full Width Promo
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+.cc-fw-promo {
+  background: var(--colour-amber-05);
+  position: relative;
+}
+
+.cc-fw-promo__image {
+  @extend %img-cover;
+  margin: 0;
+  
+  @include mq($until: sm) {
+    height: calc(220vw / 320 * 100);
```

Comment:
Could you write a comment explaining this value? I can't work out what it's doing, or why 😬

Reply:
Removed. I added it initially to force the image to display in a fixed aspect ratio but it's probably not that important